package.json

```diff
@@ -54,35 +52,37 @@
     "@storybook/react": "^6.1.20",
     "@storybook/storybook-deployer": "^2.8.7",
     "@storybook/theming": "^6.1.20",
-    "@testing-library/jest-dom": "^5.11.3",
-    "@testing-library/react": "^10.0.2",
-    "@testing-library/user-event": "^10.0.2",
+    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/react": "^11.2.5",
+    "@testing-library/user-event": "^12.8.0",
     "@types/classnames": "^2.2.10",
     "@types/jest": "^25.2.1",
-    "@types/react": "^16.9.34",
-    "@types/react-dom": "^16.9.6",
-    "@types/react-test-renderer": "^16.0.1",
+    "@types/puppeteer": "^5.4.3",
+    "@types/react": "^17.0.1",
+    "@types/react-dom": "^17.0.1",
+    "@types/react-test-renderer": "^17.0.1",
     "@types/resize-observer-browser": "^0.1.3",
     "@types/styled-components": "^5.1.0",
     "@types/styled-system": "^5.1.9",
     "@types/testing-library__react": "^10.0.1",
     "@types/throttle-debounce": "^2.1.0",
-    "amazon-chime-sdk-js": "^2.4.1",
+    "@types/uuid": "^7.0.3",
     "add": "^2.0.6",
+    "amazon-chime-sdk-js": "^2.5.0",
     "babel-loader": "^8.1.0",
     "css-loader": "^2.1.1",
     "jest": "^26.6.3",
-    "jest-image-snapshot": "^3.0.1",
-    "jest-puppeteer-docker": "^1.3.2",
+    "jest-image-snapshot": "^4.4.0",
+    "jest-puppeteer-docker": "^1.4.2",
     "mini-css-extract-plugin": "^0.7.0",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.6.0",
     "prompt-sync": "^4.2.0",
     "puppeteer": "^2.1.1",
-    "react": "^16.8.6",
+    "react": "^17.0.1",
     "react-docgen-typescript-loader": "^3.7.2",
-    "react-dom": "^16.13.1",
-    "react-is": "^16.13.1",
+    "react-dom": "^17.0.1",
```

Comment:
Should we check running the demo app and storybook since we are bumping react from v16 -> v17 which a major version change?

Reply:
Yes — or at least, do so before a release. I was doing this as a Sunday favor, so I did not.